code/backend/pages/03_Delete_Data.py

```diff
@@ -40,8 +40,8 @@
     search_handler = Search.get_search_handler(env_helper)
     search_client = search_handler.search_client
 
-    results = search_handler.get_files()
-    if results.get_count() == 0:
+    results = search_handler.get_files() if search_client is not None else None
```

Comment:
Again - could this be pushed down to the search_handler.get_file_count() which will make this even simpler

Reply:
the files from `get_files()` is being used in the next line `files = search_handler.output_results(results)`
I have removed the search_client check as it being checked in the `get_files` method